hosted_control_planes/hcp-authentication-authorization.adoc

```diff
@@ -9,9 +9,39 @@ toc::[]
 The {product-title} control plane includes a built-in OAuth server. You can obtain OAuth access tokens to authenticate to the {product-title} API. After you create your hosted cluster, you can configure OAuth by specifying an identity provider.
 
 include::modules/hcp-configuring-oauth.adoc[leveloffset=+1]
+
 include::modules/hcp-configuring-oauth-console.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
 
 * To know more about supported identity providers, see xref:../authentication/understanding-identity-provider.adoc#understanding-identity-provider["Understanding identity provider configuration"] in _Authentication and authorization_.
+
+[id="hcp-ccoctl-aws-sts_{context}"]
+== Configuring the CCO utility for a hosted cluster on {aws-short}
+
+You can configure the Cloud Credential Operator (CCO) utility (`ccoctl`) on hosted clusters to assign components IAM roles that provide short-term, limited-privilege security credentials.
+
+To configure the CCO utility for a hosted cluster, your hosted cluster must use the {aws-short} {sts-short}.
```

Comment:
For my education: are hosted clusters using AWS STS, or just manual CCO by itself? I ask because I wasn't aware that we could have AWS STS without the `ccoctl` :slightly_smiling_face:

Reply:
Let me try to explain in my comment next.